api/v1alpha2/machine_types.go

```diff
@@ -37,9 +37,8 @@ const (
 
 // MachineSpec defines the desired state of Machine
 type MachineSpec struct {
-	// ObjectMeta will autopopulate the Node created. Use this to
-	// indicate what labels, annotations, name prefix, etc., should be used
-	// when creating the Node.
+	// ObjectMeta specifies the metadata to associate with a Machine
```

Comment:
Do you want to clarify which field(s) are used?

Reply:
yes

Reply:
It looks like the CRDs need to be regenerated, they don't show the updated content